src/plugins/editor-autosuggest/spec-selectors.js

```diff
@@ -1,5 +1,6 @@
 import { createSelector } from "reselect"
 import { Set, Map } from "immutable"
+import { escapeJsonPointerToken } from "../refs-util"
```

Comment:
@shockey Is this the preferred way for shared code? Or should it go through the magic plugin system?

Reply:
This seems fine to me - the plugin system is great for exposing code that someone else may want to override/customize, but I can't imagine someone wanting to write a custom JSON Pointer escaper 😄